libs/fluid-elements/switch/src/lib/switch.spec.ts

```diff
@@ -112,6 +112,42 @@ describe('Fluid switch', () => {
     });
   });
 
+  describe('aria-checked attribute', () => {
+    it('should have aria-checked set to false by default', () => {
+      expect(fixture.hasAttribute('aria-checked')).toBeTruthy();
+      expect(fixture.getAttribute('aria-checked')).toBe('false');
+    });
+
+    it('should have aria-checked set to false when the switch is not checked', async () => {
+      fixture.checked = false;
+      await tick();
+      expect(fixture.hasAttribute('aria-checked')).toBeTruthy();
+      expect(fixture.getAttribute('aria-checked')).toBe('false');
+    });
+
+    it('should have aria-checked set to true when the switch is checked', async () => {
+      fixture.checked = true;
+      await tick();
+      expect(fixture.hasAttribute('aria-checked')).toBeTruthy();
+      expect(fixture.getAttribute('aria-checked')).toBe('true');
+    });
+
+    it('should have aria-checked set to true when the switch is clicked', async () => {
+      const label = fixture.shadowRoot?.querySelector('label');
```

Comment:
NIT: then you don't need the `label?.click()`

```suggestion
      const label = fixture.shadowRoot?.querySelector('label') as HTMLLabelElement;
```